### PR TITLE
Add GetMessageTemplateProperties option

### LINKED ISSUE #174: Allow customization of default properties collected by UseSerilogRequestLogging()
I suggest to change `Elapsed` to `ElapsedMilliseconds` by the following reasons:

1. In case of using structured logging, you need to look into message to understand the value is specified in milliseconds.
2. It will be aligned with other places: `HttpClient` logs, default `AspNetCore` logs

### PR DESCRIPTION
**Pull request by dnperfors**
Fixes #174

### CODE REVIEW

**Comment by nblumhardt** [comment]
Looks good, thank you David :+1:

**Comment by PeterZaal** [comment]
Hi @nblumhardt any news on when this will be available in a new release?

**Comment by nblumhardt** [comment]
I've created the release PR now, @PeterZaal  - thanks for the nudge! https://github.com/serilog/serilog-aspnetcore/pull/310

Have you been using this successfully via the `-dev` builds?

**Comment by PeterZaal** [comment]
Hi @nblumhardt I am on the same team as @dnperfors who created this PR, and we use a copy of the modified files in our code, which works perfectly fine!

File: src/Serilog.AspNetCore/AspNetCore/RequestLoggingOptions.cs [hunk 1]
```diff
@@ -34,6 +35,15 @@ public class RequestLoggingOptions
...
                     ? LogEventLevel.Error
                     : LogEventLevel.Information;
+        
+        IEnumerable<LogEventProperty> DefaultGetMessageTemplateProperties(HttpContext httpContext, string requestPath, double elapsedMs, int statusCode) =>
+            new[]
+            {
+                new LogEventProperty("RequestMethod", new ScalarValue(httpContext.Request.Method)),
+                new LogEventProperty("RequestPath", new ScalarValue(requestPath)),
+                new LogEventProperty("StatusCode", new ScalarValue(statusCode)),
+                new LogEventProperty("Elapsed", new ScalarValue(elapsedMs))
```
Comment (nikolay-badin-dataart) [inline src/Serilog.AspNetCore/AspNetCore/RequestLoggingOptions.cs:45]: for search requests we already have another name of the variable:
"Search request {OfferSearchId} finished processing in {RequestStatus} after {ElapsedMilliseconds:N0} ms"
should we change it there to Elapsed as well?

**Pull request merged**